branding/EULA

```diff
@@ -1,8 +1,8 @@
 The programs included with the XCP-ng system are free software.
-To know more about the licenses of all programs,
-please read each COPYING file in the /usr/share/doc/* folders.
+
+To know more about the licenses of all programs, please read each COPYING file in the /usr/share/doc/* folders.
 
 XCP-ng comes with no warranty, to the extent permitted by applicable law.
 
-For more info on XCP-ng, please check https://xcp-ng.org
-Pro support for production environment is available on https://xcp-ng.com
+For more info on XCP-ng, please check https://xcp-ng.org/.
+Pro support for production environment is available on https://vates.tech/xcp-ng/.
```

Comment:
As we cannot change that often, it's a lot safer to just use https://vates.tech

Ideally, same as my previous comment, no trailing slash (or at least no dot)

Reply:
Maybe Yann added the slash to protect the URL from the dot. A sentence with a dot is better, but the only solution I can see here is to rephrase so that the URL is not last in the sentence.